### PR TITLE
Update sites.json

### PR DESCRIPTION
**Pull request by sbritorodr**
added at lin 5194 ncbi.nlm.nih.gov site. Don't quite know if it fits on hard or impossible

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -5191,7 +5191,16 @@
             "support.native-instruments.com"
         ]
     },
-
+    {
+        "name": "NCBI",
+        "url": "https://www.ncbi.nlm.nih.gov/books/NBK3842/",
```
Comment (tupaschoal) [inline _data/sites.json]: Do you have the link saying that they're automatically deleted after inactivity? I couldn't tell if this is.

File: _data/sites.json [hunk 2]
```diff
@@ -5191,7 +5191,16 @@
...
+        "difficulty": "impossible",
```
Comment (tupaschoal) [inline _data/sites.json]: I'd say it is hard. Though it takes time it is still done after a while

File: _data/sites.json (hunk 1 above)
Comment (sbritorodr) [reply _data/sites.json]: Yes, it's in this url: https://www.ncbi.nlm.nih.gov/books/NBK3842/
"Accounts that do not have any activity in two years will be deleted"

File: _data/sites.json (hunk 2 above)
Comment (sbritorodr) [reply _data/sites.json]: okay, I'm gonna change that line

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**